### PR TITLE
doc-stacks: add inital overview of stacks

### PR DESCRIPTION
**Pull request by mhamwala**

### CODE REVIEW

**Comment by helenmasters** [comment]
There are a few typos in the first section (sorry I couldn't add these comments as those lines were unchanged).

Line 8
- enables -> enable
- typo "development"

Line 9 
- additonal libraries an tools -> additional libraries and tools
- required simplify -> required to simplify
- best practise -> best practice
- "They are often created by technology experts" - can we say this? Can't we just lose that bit and say "Stacks are an easy way to achieve consistency and adopt best practices across many applications."

Line 10 
- utilises -> utilizes

File: docs/stacks/stacks-overview.md [hunk 1]
```diff
@@ -1,8 +1,43 @@
...
+Stacks are catagorised as either `stable`, `incubator` or `experimental` depending on the content of the stack.
+
+- `stable/`: Stable stacks meet a set of technical requirements which are yet to be defined.
+
+- `incubator/`: The stacks in the incubator folder are activly been worked upon to satisfy the stable critria.
+
+- `experimental/`: Exprimental stacks are not been actively been worked upon and  may not fulfill the requirements of an Appsody stack. These can be used for trying out specific capabilites or proof of concept work.
+
+## Getting started
+Follow the [quick start guide](../getting-started/quick-start.md) to get you up and running with Apposdy stacks.
```
Comment (helenmasters) [inline docs/stacks/stacks-overview.md]: "Follow our [Quick Start Guide](../getting-started/quick-start.md) to quickly get up and running with Appsody."

File: docs/stacks/stacks-overview.md [hunk 2]
```diff
@@ -1,8 +1,43 @@
...
+
+To explore existing stacks that are available to you:
```
Comment (helenmasters) [inline docs/stacks/stacks-overview.md]: explore suggests that there is a lot of information somewhere... how about something more like

"To find a list of the existing stacks that are available for you to use:"

File: docs/stacks/stacks-overview.md [hunk 3]
```diff
@@ -1,8 +1,43 @@
...
+- Visit the [Appsody website](https://appsody.dev) or
+- Run `appsody list` command using our CLI
```
Comment (helenmasters) [inline docs/stacks/stacks-overview.md]: Run the `appsody list` command within the appsody CLI

Comment (mhamwala) [reply docs/stacks/stacks-overview.md]: how does this sound?
Run the `appsody list` command when using the Apposdy CLI

Comment (mhamwala) [reply docs/stacks/stacks-overview.md]: or 
Use the `appsody list` command within the Apposdy CLI

File: docs/stacks/stacks-overview.md [hunk 4]
```diff
@@ -1,8 +1,43 @@
...
+
+To learn how to go about modifying an existing stack go to [modifying a stack](create-or-modify.md#modifying-a-stack).
+
+## Creating new stacks
+We are actively working to create new stacks so that more people can adopt Appsody. If you find that none of the existing stacks meet your needs please reach out to us on the [Appsody slack]() or create a new GitHub issue to track the discussion.
+
+We always welcome any contributions. If you wanted to create your own stack for a framework or language that we do not currently support, please review the [contributing guidelines](../../CONTRIBUTING.md) and follow the steps outlined in [creating a stack](create-or-modify.md#creating-a-stack).
+
+## Need help?
+If you have a question that you can't find an answer to, we would also like to hear about that too. You can also reach out to the community for assistance on [Slack]().
```
Comment (helenmasters) [inline docs/stacks/stacks-overview.md]: You can also reach -> You can reach

File: docs/stacks/stacks-overview.md [hunk 5]
```diff
@@ -1,8 +1,43 @@
...
-- Stable -->
\ No newline at end of file
+Appsody provides pre-configured application stacks that enables rapid devlopement of quality microservice-based applications. Stacks include a base container image and project templates which act as a starting point for your application development.
+
+Appsody stacks include language runtimes, frameworks and any additonal libraries an tools that are required simplify your local application development. They are often created by technology experts and are an easy way to manage consistency and adopt best practises across many applications.
+
+**Template:** A template utilises the base image and provides a starter application that is ready to use. It leverages existing capabilities provided by that image and can extend functionality to meet your application requirements.
+
+---
+Stacks are catagorised as either `stable`, `incubator` or `experimental` depending on the content of the stack.
```
Comment (helenmasters) [inline docs/stacks/stacks-overview.md]: catagorized

File: docs/stacks/stacks-overview.md [hunk 6]
```diff
@@ -1,8 +1,43 @@
...
+
+- `stable/`: Stable stacks meet a set of technical requirements which are yet to be defined.
+
+- `incubator/`: The stacks in the incubator folder are activly been worked upon to satisfy the stable critria.
```
Comment (helenmasters) [inline docs/stacks/stacks-overview.md]: activly been worked upon -> actively being worked on

File: docs/stacks/stacks-overview.md [hunk 7]
```diff
@@ -1,8 +1,43 @@
...
+
+- `experimental/`: Exprimental stacks are not been actively been worked upon and  may not fulfill the requirements of an Appsody stack. These can be used for trying out specific capabilites or proof of concept work.
```
Comment (helenmasters) [inline docs/stacks/stacks-overview.md]: I still don't like this wording but let's at least fix the typos ->
"Experimental stacks are not being actively worked on and may not fulfill the requirements of an Appsody stack."

**Pull request merged**